spec/forms/state_file/nj_homeowner_property_tax_worksheet_form_spec.rb

```diff
@@ -0,0 +1,46 @@
+require "rails_helper"
+
+RSpec.describe StateFile::NjHomeownerPropertyTaxWorksheetForm do
+  let(:intake) { create :state_file_nj_intake }
+
+  describe "validations" do
+    let(:form) { described_class.new(intake, params) }
+
+    it_behaves_like :nj_money_field_concern, field: :property_tax_paid, can_be_empty: true, must_be_positive: true do
+      let(:form_params) do
+        { property_tax_paid: money_field_value }
+      end
+    end
+
+    context "cannot be zero" do
+      let(:params) do
+        { :property_tax_paid => 0 }
+      end
+
+      it "is invalid" do
+        expect(form.valid?).to eq false
+        expect(form.errors[:property_tax_paid]).to include "must be greater than or equal to 1"
```

Comment:
[pebble] this can be deleted, it's handled by the `must_be_positive` true case in the nj_money_field test

Reply:
same comment in other test for tenant